streams/src/test/java/org/apache/kafka/streams/integration/StandbyTaskEOSIntegrationTest.java

```diff
@@ -360,11 +368,9 @@ public KeyValue<Integer, Integer> transform(final Integer key, final Integer val
                             throw new RuntimeException("Injected test error");
                         }
 
-                        return KeyValue.pair(key, value);
+                        store.put(key, value);
```

Comment:
We do we add a put here?

Reply:
Similar to the return null replacement with delete. Since we would return a pair with key and value, now I put it in store.